### DISCUSSION TITLE
Password Invalid

### DISCUSSION THREAD

**Discussion by Amahro** [iotaledger/firefly#7603] [Help & Support]
I'm not able to enter at Firefly site because my password isn't accepted anymore.

**Comment by begonaalvarezd** [comment]
Hi, do you mean the pin or the stronghold password? 
If you cant access anymore to your profile you can always recover it creating a new one using your 24 words, a stronghold file or your ledger device

**Comment by Amahro** [comment]
Thank you!
Have a nice day!

Em ter., 17 de out. de 2023 04:35, Begoña Álvarez de la Cruz <
***@***.***> escreveu:

**Comment by Amahro** [comment]
Thanks a lot.

Em ter., 17 de out. de 2023 06:11, Begoña Álvarez de la Cruz <
***@***.***> escreveu:
